### DISCUSSION TITLE
How to configure the color of a category?

### DISCUSSION THREAD

**Discussion by Watcher24** [allure-framework/allure2#1987] [Questions & Support]
We have successfully defined new categories using [categories.json](https://git.magnolia-cms.com/projects/BUILD/repos/aws-build-scripts/raw/allure-configuration-files/categories.json) - which we add to src/test/resources before triggering the test-scenarios with maven.

Now we would like to define distinct colors for all the categories - this would allow to see & distinguish the errors at 1st sight +/- by only looking to the report overview page. Currently we get grey for both "DisabledTests" and "SessionNotCreatedException" - which are two fundamentally different cases. iirc also grey for "WebDriverIllegalStateException" - this actually let us think, that all "failure(s)" occurring in cucumber-Before-Hooks appear grey ... but still this could be just randomly, idk.
 
So ... **is there a way to specify the colors for the categories?** 
(I could not find anything in docu, stackoverflow, etc. ... and what we attempted failed so far.)

**Comment by mdaraz** [comment]
Hi, this is good question. I'd like to do the same thing and have no idea how I can do it. Any tip or suggestion?

**Comment by Watcher24** [comment]
Hello @baev 
I dare to address you here, since I guess you are the main contributer / developer on allure-server.

Is it already possible to define the color of a category?

If not, I consider to find a way to implement it and contribute it. 
In that case, you may want to give me a pointer where I ideally would do that?
